sortingUtils/sort.js

```diff
@@ -0,0 +1,19 @@
+const comparingNumbersFunction = require("./compareNumbers");
+
+module.exports = arrayToSort => {
+  const arrayOfNumbers = [];
+  const arrayOfStrings = [];
+
+  arrayToSort.forEach(element => {
```

Comment:
You could do this more functional but on this level it is acceptable

Reply:
Added filter functions for extracting numbers and string separately. Maybe that will be better solution.